README.md

```diff
@@ -1,54 +1,195 @@
 # BizHawk
 
-BizHawk is a multi-system emulator written in C#. BizHawk provides nice features for casual gamers such as fullscreen, rewind, and joypad support in addition to rerecording and debugging tools for all system cores.
+[![GitHub latest release](https://img.shields.io/github/release/TASVideos/BizHawk.svg?logo=github&logoColor=333333&style=popout)](https://github.com/TASVideos/BizHawk/releases/latest)
+[![dev builds | AppVeyor](https://img.shields.io/badge/dev_builds-AppVeyor-orange.svg?logo=appveyor&logoColor=333333&style=popout)](https://ci.appveyor.com/project/zeromus/bizhawk-udexo/history)
+[![unique systems emulated | 22](https://img.shields.io/badge/unique_systems_emulated-22-darkgreen.svg?logo=buffer&logoColor=333333&style=popout)](https://github.com/TASVideos/BizHawk/blob/master/README.md#features)
 
-[Release Notes](http://tasvideos.org/Bizhawk/ReleaseHistory.html)
+A multi-system emulator written in C#. As well as quality-of-life features for casual players, it also has recording/playback and debugging tools, making it the first choice for TASers (Tool-Assisted Speedrunners).
 
-[Here](http://tasvideos.org/Bizhawk/Features.html) is a list of features offered by bizhawk.
+Click the "release" button above to grab the latest stable version. New users on Windows will need [this](https://github.com/TASVideos/BizHawk-Prereqs/releases/latest). The changelog is [here at TASVideos](http://tasvideos.org/Bizhawk/ReleaseHistory.html). **Never mix different versions** of BizHawk — Keep each version in its own folder.
```

Comment:
LOL come on, promote prereqs for real. Add an icon for it, people still miss it even with it being on top of every page. And when mentioning it here, also link to details on what it contains, below.

Reply:
good idea